### PR TITLE
Added syntax highlighting

### PR DESCRIPTION
**Pull request by exdis**
Highlight results in console

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -88,7 +89,8 @@ function processStream(options) {
         if (options.outputFile) {
             fs.writeFileSync(options.outputFile, serializedResult, 'utf-8');
         } else {
-            console.log(serializedResult);
+            const result = options.noColor ? serializeResult : colorize(serializedResult);
```
Comment (lahmatiy) [inline index.js]: should be `options.color` – clap removes `no-` and inverts value

Comment (exdis) [reply index.js]: fixed

File: utils/colorize.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+const { TOKEN_COLORS } = require('./constants');
+const tokenize = require('./tokenize');
+
+const markKeys = (tokens) => {
+    for (let i = 0; i < tokens.length; i++) {
+        let token = tokens[i];
+
+        if (token.type === STRING) {
+            for (let j = i + 1; j < tokens.length; j++) {
+                if (tokens[j].type === WHITESPACE) {
```
Comment (lahmatiy) [inline utils/colorize.js]: `switch/case` would be better here

File: test/test.js [hunk 3]
```diff
@@ -74,7 +91,7 @@ it('should output version', () =>
 );
 
 it('should read content from stdin if no file specified', () =>
-    run('version')
+    run('version', '--no-color')
```
Comment (lahmatiy) [inline test/test.js]: missed `--no-color`

**Pull request merged**